library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -1981,6 +1981,12 @@ libspdm_return_t libspdm_acquire_sender_buffer (
     }
     spdm_context->sender_buffer = *msg_buf_ptr;
     spdm_context->sender_buffer_size = *max_msg_size;
+    #if LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP
```

Comment:
Ok. I can understand why put this here, so that you wouldn't need to modify all the different requester command functions to use either the sender buffer or the scratch space. I think it's ok to get it working like this, but may be confusing to follow for someone first looking at the code or revisiting it after a while. Maybe in the future, as part of a refactoring or something, it would be more straight forward to have the decision in each of the requester functions. Just a thought.

Reply:
that is right. I only do necessary change to make it work in Q3, due to limited time.
We may do refactoring at https://github.com/DMTF/libspdm/issues/1213 in Q4.

Reply:
Sounds good.